.github/workflows/release.yml

```diff
@@ -5,12 +5,43 @@ on:
     tags: ["v*"]
 
 jobs:
-  github_release:
+  log-updates:
+    name: Log packages to publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the new tag
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: This release will publish the following packages
+        run: git diff --name-only HEAD^..HEAD
+
+  npm-release:
+    name: Publish release on npm
+    runs-on: ubuntu-latest
+    needs: log-updates
+    environment: npm
+    steps:
+      - name: Checkout the new tag
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Q: Is fetch depth 0-based or 1-based?

Reply:
It's `1`-based, `0` means "fetch all". We need it in order to find the previous tag.